developer/openshift/dev_setup.sh

```diff
@@ -232,23 +240,18 @@ install_pipeline_service() {
     --kustomization "git::$GIT_URL/developer/openshift/gitops/argocd?ref=$GIT_REF" |
     indent 2
 
+  # Render the content of the app of apps and change the url/branch to target the
+  # expected repository/branch
+  manifest_dir="$(find "$WORK_DIR/environment/compute" -mindepth 1 -maxdepth 1 -type d)"
+  kubectl create -k "$manifest_dir" --dry-run=client -o yaml >"$manifest_dir/pipeline-service.yaml"
+  yq -i ".spec.source.repoURL=\"$GIT_URL\" | .spec.source.targetRevision=\"$GIT_REF\"" "$manifest_dir/pipeline-service.yaml"
```

Comment:
This is going to replace source in all argo apps right ?  This may be an issue if we start using external argocd apps like in https://github.com/openshift-pipelines/pipeline-service/pull/585 (where we use a Helm chart as source of the argo app).
So we need to replace only sources of apps pointing to the `pipeline-service` repository that's why I am grepping for such apps here https://github.com/openshift-pipelines/pipeline-service/pull/588/files#diff-342e128bf9808c7ef44522ed6360d1b5f838cc9308f916b6c4489300abfc09cbR162

Reply:
That's correct. I've updated the PR to use kustomize patch mechanism to set the applications with the proper source.

Reply:
I'll mark the conversation as resolved. Let me know if you do not agree with the latest solution.